### PR TITLE
Call API

### PR DESCRIPTION
**Pull request by avbel**
Also with Bridge API

### CODE REVIEW

**Comment by avbel** [comment]
I left 1 PR with Bridge and Call API together to avoid merge conflict. All new branched I will create based on `v3-preview`

File: src/Bandwidth.Net/Api/Bridge.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    /// </summary>
+    /// <param name="query">Optional query parameters</param>
+    /// <param name="cancellationToken">>Optional token to cancel async operation</param>
+    /// <returns>Collection with <see cref="Bridge" /> instances</returns>
+    /// <example>
+    ///   <code>
+    /// var bridges = client.Bridge.List(); 
+    /// </code>
+    /// </example>
+    IEnumerable<Bridge> List(BridgeQuery query = null,
```
Comment (dtolb) [inline src/Bandwidth.Net/Api/Bridge.cs]: is this async? the `>Optional token to cancel async operation` makes me think, yes? but the example doesn't `await`

File: src/Bandwidth.Net/Api/Bridge.cs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    /// </summary>
+    /// <param name="bridgeId">Id of bridge to get calls</param>
+    /// <param name="cancellationToken">>Optional token to cancel async operation</param>
+    /// <returns>Collection with <see cref="Call" /> instances</returns>
+    /// <example>
+    ///   <code>
+    /// var calls = client.Bridge.GetCalls(); 
+    /// </code>
+    /// </example>
+    IEnumerable<Call> GetCalls(string bridgeId, CancellationToken? cancellationToken = null);
```
Comment (dtolb) [inline src/Bandwidth.Net/Api/Bridge.cs]: Same thing as above, ` <param name="cancellationToken">>Optional token to cancel async operation</param>` has async in the description

File: src/Bandwidth.Net/Api/Call.cs [hunk 3]
```diff
@@ -0,0 +1,939 @@
...
+    /// </summary>
+    /// <param name="query">Optional query parameters</param>
+    /// <param name="cancellationToken">>Optional token to cancel async operation</param>
+    /// <returns>Collection with <see cref="Call" /> instances</returns>
+    /// <example>
+    ///   <code>
+    /// var calls = client.Call.List(); 
+    /// </code>
+    /// </example>
+    IEnumerable<Call> List(CallQuery query = null,
```
Comment (dtolb) [inline src/Bandwidth.Net/Api/Call.cs]: Same as bridges, looking at the code, it seems that it is Async. is there a reason that `List` is just `List` and not `ListAsync`?

Comment (avbel) [inline src/Bandwidth.Net/Api/Call.cs]: `List` methods are not async. They return lazy collection instance immediately. When we enum this collection async request will be excuted

**Pull request merged**